components/support/locale/src/main/java/mozilla/components/support/locale/LocaleManager.kt

```diff
@@ -73,7 +73,7 @@ object LocaleManager {
      */
     fun getSystemDefault(): Locale {
         val config = Resources.getSystem().configuration
-        return ConfigurationCompat.getLocales(config).get(0)
+        return ConfigurationCompat.getLocales(config).get(0)!!
```

Comment:
I think we should find a way to prevent **!!** that could let to crashes

Reply:
Maybe we could fallback to `Locale.getDefault()`

Reply:
Done!

Reply:
👍🏽